docs/getting-started/what-is-dotnet.md

```diff
@@ -44,9 +44,9 @@ Starting with Windows 8, a version of the .NET Framework is installed as an oper
 </tr>
 </table>
 
-Although the .NET Framework and .NET Core target different platforms and represent different approaches to application development and deployment, they both conform to the .NET Standard 1.3. This means that they offer a high degree of compatibility and identical behavior between one another. In particular:
+Although the .NET Framework and .NET Core target different platforms and represent different approaches to application development and deployment, they both conform to the [.NET Standard 1.3](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md ".NET Standard 1.3"). This means that they offer a high degree of compatibility and identical behavior between one another. In particular:
     
-- Their high degree of compatibility is makes it easy for .NET Framework developers to develop applications with .NET Core that target different platforms and devices.
+- Their high degree of compatibility makes it easy for .NET Framework developers to develop applications with .NET Core that target different platforms and devices.
 - Their high degree of compatibility also makes it easy for .NET Core developers to develop applications with the .NET Framework that target Windows desktop, tablet, and phone.
 - Libraries written for the .NET Framework or .NET Core can easily be made to work on the other platform. 
```

Comment:
47: Substitute "to" for "between" ?
49, 50: Parallel introductions seem clunky; merge into a single idea, that's it's easy to transfer both code and developer skills between Framework and Core?